src/jsonld/corporateContact.tsx

```diff
@@ -1,12 +1,25 @@
-import React from 'react';
+import * as React from 'react';
 import Head from 'next/head';
 
 import markup from '../utils/markup';
 
-const formatIfArray = value =>
+export interface ContactPoint {
```

Comment:
We can leave this and the below interface not exported.

Reply:
It's advisable to export the interface so that the end user can be able to access it. It can be used to offer type checking. For example maybe a user wants to generate a config file for contactPoint they can use the interface for type checking.

```javascript
import { ContactPoint, CorporateContactJsonLd } from 'next-seo'

const contactPoint: ContactPoint = {[
  {
    telephone: '+1-401-555-1212',
    contactType: 'customer service',
    areaServed: 'US',
    availableLanguage: ['English', 'Spanish', 'French'],
  },
]}

<CorporateContactJsonLd
  url="http://www.your-company-site.com"
  logo="http://www.example.com/logo.png"
  contactPoint={contactPoint}
/>
```

Reply:
Of course! Makes perfect sense. 😅

Reply:
Sweet! I will update the PR to export interface